src/controllers/users/controller.ts

```diff
@@ -34,7 +38,6 @@ class UserController {
         }
```

Comment:
use destructuring. name variable userList instead extractedData. skip, limit, sort passed into options argument, count should be calculated outside response

Reply:
Used Destructuring. updated name variable from extractedData to userList and passed skip, limit, sort into options argument.count should be in outside of the response.